### PR TITLE
Update Get-SmbShare.md for scopename and related examples

### PR DESCRIPTION
**Pull request by nedpyle**
# PR Summary

Redesigned help to not be misleading about scopename vs cimsession and added a remote server example: previous examples and help were often referring to failover clustering without being explicit, showing output that 99,9% of customers will never see because they are not running clusters. I am going to do this against get-smbshare, new-smbshare, set-smbshare, remove-smbshare as needed.

## PR Checklist

- [X] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [X] **Summary:** This PR's summary describes the scope and intent of the change.
- [X] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [X] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docset/winserver2022-ps/smbshare/Get-SmbShare.md [hunk 1]
```diff
@@ -65,21 +50,29 @@ This command retrieves the SMB shares on the computer.
...
 Name                          ScopeName                     Path                          Description 
 ----                          ---------                     ----                          ----------- 
-VMS1                          Contoso-FS                    I:\VMS
+VMS1                          *                             I:\VMS
 ```
 
 This command retrieves information about the SMB share named 'VMS1' on the local computer.
 
-### Example 3: Display information about the SMB share named 'VMS1' on the local computer in a list
+### Example 3: Display information about the SMB shares on a remote computer.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/smbshare/Get-SmbShare.md:58]:
```suggestion
### Example 3: Display information about the SMB shares on a remote computer
```

File: docset/winserver2022-ps/smbshare/Get-SmbShare.md [hunk 2]
```diff
@@ -342,7 +335,7 @@ Accept wildcard characters: False
 ```
 
 ### -ScopeName
-Specifies the scope of the share by name.
+Specifies the scope of the share by name. For use with Windows Server failover cluster file server resources.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/smbshare/Get-SmbShare.md:338]:
```suggestion
Specifies the scope of the share by name for use with Windows Server failover cluster file server resources.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @nedpyle, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]
Can you please check if this change can be applied to other OS versions of the cmdlet's article? Thank you Ned

**Comment by nedpyle** [comment]
> Can you please check if this change can be applied to other OS versions of the cmdlet's article? Thank you Ned

yes, all previous versions

**Comment by get-itips** [comment]
Can you please make also those edits in this PR? Thank you

**Comment by nedpyle** [comment]
> Can you please make also those edits in this PR? Thank you

What edits?

**Comment by get-itips** [comment]
> > Can you please make also those edits in this PR? Thank you
> 
> What edits?

I am referring to these other files

image

**Comment by nedpyle** [comment]
> > > Can you please make also those edits in this PR? Thank you
> > 
> > 
> > What edits?
> 
> I am referring to these other files
> 
> image

I'm not doing that, there are too many edits in too many files. They should be bulk fixed, not individually edited.

**Pull request merged**